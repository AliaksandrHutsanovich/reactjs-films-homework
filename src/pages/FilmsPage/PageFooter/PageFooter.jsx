import React, { memo } from 'react';
import { Layout } from 'antd';
import { HeadLine, Text } from '../../../components';

import styles from '../FilmsPage.scss';

const { Footer } = Layout;

const PageFooter = () => (
  <Footer className={styles.footer}>
    <HeadLine text="FILMS" level={3} fontFamily="arial" type="blue" align="center" />
    <Text type="normal">Copyright Â© 2018 FILMS Sanderas</Text>
  </Footer>
);

export default memo(PageFooter);
