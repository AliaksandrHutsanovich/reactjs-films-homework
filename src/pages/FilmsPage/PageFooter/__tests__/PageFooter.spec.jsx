import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import PageFooter from '../PageFooter';

describe('<PageFooter />', () => {
  const renderer = new ShallowRenderer();

  it('should render properly', () => {
    renderer.render(<PageFooter />);
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
});
