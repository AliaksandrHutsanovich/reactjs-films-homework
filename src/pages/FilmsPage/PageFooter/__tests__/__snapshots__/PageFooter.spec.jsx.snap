// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageFooter /> should render properly 1`] = `
<Footer
  className="footer"
>
  <UNDEFINED
    align="center"
    fontFamily="arial"
    level={3}
    text="FILMS"
    type="blue"
  />
  <UNDEFINED
    type="normal"
  >
    Copyright Â© 2018 FILMS Sanderas
  </UNDEFINED>
</Footer>
`;
